source/mainnet/smart-contracts/tutorials/voting/voting-dapp.rst

```diff
@@ -37,116 +39,109 @@ need for the user to host their own Concordium node.
 |bw|
 -------------------------
 
-A web wallet is a piece of code that can be added as an extension to supported browsers such as ``Chrome``.
-The web wallet allows you to interact with the chain and make transactions.
-Currently, the |bw| does this by connecting to a (gRPC) server that communicates with a node.
-The |bw| hosts the private keys corresponding to the accounts of the user.
+Concordium's browser wallet is an extension to Chromium-based browsers such as Google Chrome.
```

Comment:
everywhere in the documentation:

```suggestion
Concordium's |bw| is an extension to Chromium-based browsers such as Google Chrome.
```

Reply:
This will expand to "Concordium's Concordium Wallet for Web", but even fixing the double Concordium, I think the long title just sounds super unnatural in a casual sentence like this.